### DISCUSSION TITLE
VS Code

### DISCUSSION THREAD

**Discussion by DthArgelius** [Launch-X-Latam/MisionFrontEnd#9] [Q&A]
Para los que usaremos Visual Studio Code, cual o cuales seriían las mejores herramientas o complementos para mejorar la experiencia con el software

**Comment by Eduardo282x** [comment]
Yo te recomiendo que descargues las extensiones basicas para HTML, CSS, JavaScript, tambien puedes agregar adicional la extensión de Live Server para ver los cambios que realices en la aplicación en tiempo real cada ves que guardes en tu navegador

**Comment by magouwuxd** [comment]
depende que vaya a decir el ser supremo, pero ve instalando las extensiones de prettier, emmet, error lens. Esos uso yo, ya si usa React ES7+React/redux/react-native snippets y de vue busca otro de snippets. Eslint tambien es bueno pero es muy complicado de usar

**Comment by RocioSanchez2022** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/114268241/192554458-eea6a51c-a30d-4b2f-99ae-31d77674bab6.png)
![Screenshot_2](https://user-images.githubusercontent.com/114268241/192554461-9f58d266-0cc6-4afb-94b1-110996c3346e.png)
![Screenshot_1](https://user-images.githubusercontent.com/114268241/192554466-9357db96-d2fa-4aa5-847b-388cbc8ae3ab.png)


Yo te recomiendo estas extensiones en el visual studio code, son las que mas se usan!

**Reply by OscarGUD** [reply]
La extensión de colorear los brackets ya no es necesaria, si te sirve la puedes modificar desde Vs code.
![image](https://user-images.githubusercontent.com/100248835/192572510-53b018a8-2c99-464f-adbd-910a469e4c8d.png)

**Comment by Beather1205** [comment]
pluggins que pueda utilizar para autocompletado y facilitar el trabajo?

**Reply by OscarGUD** [reply]
Estas son las que utilizo para autocompletar si te sirven.
![image](https://user-images.githubusercontent.com/100248835/192573017-f04c1b83-91e7-40d3-a972-8d5ab3f8c676.png)

**Reply by Vguayas12** [reply]
Hola puedes utilizar la extensión de Emmet, es una de las mejores.
<img width="190" alt="image" src="https://user-images.githubusercontent.com/114126509/192578836-f9eb8708-f504-4cb3-b1c0-f6ad98a7af03.png">

**Reply by OscarGUD** [reply]
Emmet ya viene por defecto al descargar Vs code. 
https://code.visualstudio.com/docs/editor/emmet

**Comment by SilvyeCG** [comment]
Yo te recomiendo además de las que ya mencionaron:
- Office Viewer(Markdown Editor) 
- Mintlify Doc Writer
- Live Server
